lib/engine/game/g_1846/game.rb

```diff
@@ -741,6 +783,111 @@ def train_buying_power(entity)
         def ability_time_is_or_start?
           active_step.is_a?(G1846::Step::Assign) || super
         end
+
+        def little_miami
+          @little_miami ||= company_by_id('LM')
+        end
+
+        def little_miami_hexes
+          @little_miami_hexes ||= little_miami.all_abilities.first.hexes
+        end
+
+        def little_miami_router
+          @little_miami_router ||=
+            Engine::Corporation.new(name: 'LM Corp', sym: 'LM Corp', tokens: [], coordinates: little_miami_hexes.first)
+        end
+
+        def compute_little_miami_graph
+          graph = Graph.new(self, no_blocking: true, home_as_token: true)
```

Comment:
can we add !loading checks in all these graph places?

Reply:
983bf5ca5